src/commands/dev/lib/dev-router.ts

```diff
@@ -71,7 +71,8 @@ export default async function(
 
       const keys: string[] = [];
       const matcher = PCRE(`%${src}%i`, keys);
-      const match = matcher.exec(reqPathname);
+      let match = matcher.exec(reqPathname);
```

Comment:
I think you can do this in one line:

```js
let match = matcher.exec(reqPathname) || matcher.exec(`/${reqPathname}`);
```

Or perhaps even make the optional `/` part of the regex.

Reply:
Updated it to one line, I think it's good to check for this even if we did update the dynamic routing RegExp to have the beginning slash as optional.